src/main/java/at/ac/tuwien/kr/alpha/common/depgraph/Edge.java

```diff
@@ -0,0 +1,57 @@
+package at.ac.tuwien.kr.alpha.common.depgraph;
+
+/**
+ * An edge in a dependency graph.
+ */
+// TODO ensure immutability
+public class Edge {
+
+	private final Node target;
+	private final boolean sign;
+	private final String label;
```

Comment:
What is the label used for? It seems to be a `String` representation of the sign?